lib/Metrc.js

```diff
@@ -186,10 +202,13 @@ function Metrc(domain, licenseNumber, apiKey, userKey) {
   }
   
   return {
-    get: get,
+    get,
+    post,
+    put,
     post: post,
+    put: put,
```

Comment:
how did that pass the linter?

Reply:
the original code didn't have linting configured, and am hesitant about doing a full formatting pass since this is a fork (which we may merge).